mpc/coordinator/contribute.py

```diff
@@ -61,7 +61,7 @@ def contribute(
         base_url: str,
         key_file: str,
         challenge_file: str,
-        contribute: Callable[[], str],
+        contribute_cb: Callable[[], str],
```

Comment:
What does the `cb` suffix stand for? callback?

Reply:
Yes.  `cb` just means callback.  `pylint` complains about `contribute` because there is a module of that name imported.